### ISSUE TITLE
Frontend Deployment not working

### ISSUE THREAD

**Issue by waqas-syed** [aws-amplify/amplify-console#468]
**How to recreate:**
- Create an Amplify project using amplify CLI and do `amplify push`.
- After successful deployment of backend, open the deployed Amplify App and in the **Frontend Deployment** tab, select a Git provider.
- Choose the Git branch and the current backend environment which you have already deployed, along with the Service Role.
- Click **Next**
- After reviewing, click `Save and Deploy`.

In the Developer console, I am getting the following error:
`main.js:81 POST https://us-west-2.console.aws.amazon.com/amplify/api/amplify 400 (Bad Request)`
`BadRequestException: You should at least provide one valid token
    at Object.o [as extractError] (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:41:175131)
    at constructor.a (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:55:144363)
    at constructor.callListeners (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:55:147503)
    at constructor.emit (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:55:147214)
    at constructor.emitEvent (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:81:1396747)
    at constructor.e (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:81:1392389)
    at n.runTo (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:81:1445085)
    at https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:81:1445290
    at constructor.<anonymous> (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:81:1392599)
    at constructor.<anonymous> (https://d1cr1y5smcf7os.cloudfront.net/88a304120d3cc32dc09729c77067ebc7b6797ea8/main.js:81:1396802)`

It was working fine this morning. Is it just happening to me?

**Comment by Joycehao19** [comment]
@waqas-syed I can't reproduce you issue. Would you mind try it again? If you still have this issue, you can try re-set `amplify configure` + re-create `amplify init` + your custom setting + `amplify push`

**Comment by Joycehao19** [comment]
Closing for now. If you still have this issue, please reopen it. Thanks

**Comment by daltonfury42** [comment]
I observed the same issue today

**Comment by shnjp** [comment]
Same issue today

**Comment by shnjp** [comment]
I ran into this and subsequent problems, but I was able to solve them.

> You should at least provide one valid token ...

This message is probably a Token issue in the AWS Console, so I reloaded my browser and moved on.

Next Error
> "There was an issue setting up your repository. Please try again later. Failed", "errors":[{"resource": "Hook", "code ": "custom", "message": "Hook already exists on this repository"}], "documentation_url":" https://developer.github.com/v3/repos/hooks ... /#create-a-hook"})"

I removed it from the github repos settings page, probably because it looks like amplify failed to deploy halfway through and left a garbage webhook behind.

Last Error
> Invalid build spec, yaml syntax error 

Our version of `amplify.yml` seemed to be out of date, so we set it to `1.0`. I cured it and we can now move on.
Amplify didn't VALIDATE in this previous step, but in this step, it did and silently failed. And didn't check the timing of the actual CD.
